CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 0.5.1-dev0
+
+### Enhancements
+
+* Default `download_dir` now lives under `$HOME/.cache/unstructured/ingest` instead
```

Comment:
Nit: let's call out `unstructured-ingest`, so "Default `download_dir`" -> "Default `unstructured-ingest` `download_dir`."

Also maybe instead of "instead of in the root directory under the name of `tmp-ingest-...`" something like "instead of a "tmp-ingest-" dir in the the working directory"